packages/client/src/components/ToolDetailsPage/ToolDetailsPage.css

```diff
@@ -97,9 +99,31 @@ h3 {
 }
 
 .tool-description > p:first-child {
+  font-size: 1.625rem;
+  margin-bottom: 2.5rem;
+}
+
+.tool-description > p:nth-child(2) {
+  font-size: 1.375rem;
   margin-bottom: 2.5rem;
 }
 
+.tool-description > p:nth-child(3) {
```

Comment:
This is a good an smart way to do a selector and it is very good that you applied it 👍🏽 
However, I would advise using a more specific approach for such a specific thing, so it would be easier to identify and not be affected by possible changes. For example, if any new paragraph would be added to the tool description, it would break this style. Of course, that is unlikely, but not impossible 🤓 
I would do an id for that specific p and select accordingly.

Reply:
Thanks. Fixed that ✅😊